pipeline/config/exoscale/sc-config.yaml

```diff
@@ -60,3 +60,5 @@ prometheus:
   predictLinearLimit: 66
 thanos:
   enabled: true
+velero:
+  includedNamespaces: []
```

Comment:
You can remove this since it is the default config. It will be added by apps init in the pipeline.